src/Symfony/Component/Inflector/Tests/InflectorTest.php

```diff
@@ -294,30 +294,30 @@ public function pluralizeProvider()
     /**
      * @dataProvider singularizeProvider
      */
-    public function testSingularize($plural, $singular)
+    public function testSingularize($plural, $expectedSingular)
```

Comment:
Should I revert the changes on this method or backport it on 3.4?

Reply:
yes please